hl-partials/rangeinputs.html

```diff
@@ -0,0 +1,17 @@
+<!-- Range Input -->
+<div class="ampstart-input inline-block relative m0 p0 mb3 ">
+  <input type="range" value="" name="name11" id="ip11" class="block border-none p0 m0" placeholder="Select a range">
+  <label for="ip11" class="absolute top-0 right-0 bottom-0 left-0">
+    Select a range
+  </label>
+</div>
+<!-- End Input-->
+
+<!-- Disabled Range Input -->
+<div class="ampstart-input inline-block relative m0 p0 mb3 ">
+  <input type="range" value="" name="name11" id="ip11a" class="block border-none p0 m0" placeholder="Select a range" disabled>
+  <label for="ip11a" class="absolute top-0 right-0 bottom-0 left-0">
+    Select a range
+  </label>
+</div>
+<!-- End Input-->
```

Comment:
The new lines on these files are on purpose. (We dont want new lines when the code highlighting is rendered.)